svmbir/svmbir.py

```diff
@@ -425,6 +425,9 @@ def recon(sino, angles,
     else:
         init_image_value = 0
 
+    if p < 0:
```

Comment:
I think this needs to be ``if p <= 0``. 
I'm going to reject this pull request. 

Please make the suggested changes, and resubmit.